### PR TITLE
[Datacomp] Add clean_captions and filter_clip_score components

### PR DESCRIPTION
**Pull request by alexanderremmerie**

### CODE REVIEW

**Review by NielsRogge — APPROVED** [review]

**Review by RobbeSneyders — CHANGES_REQUESTED** [review]
Thanks @alexanderremmerie 

Can you just revert all the commit hashes in the component specs?

**Comment by NielsRogge** [comment]
@RobbeSneyders I've reverted the commit hashes, but I have not yet had an experience where the dev tags work out of the box. I always need to rebuild the Docker images because the dev ones don't work

**Comment by RobbeSneyders** [comment]
We might want to implement this: https://github.com/ml6team/fondant/issues/365
Also for the local runner.

File: examples/pipelines/datacomp/simple_pipeline.py [hunk 1]
```diff
@@ -16,8 +16,8 @@
     pipeline_name="datacomp-filtering",
     pipeline_description="A pipeline for filtering the Datacomp dataset",
     base_path=PipelineConfigs.BASE_PATH,
+    # base_path="/Users/nielsrogge/Documents/fondant_artifacts_datacomp",
```
Comment (RobbeSneyders) [inline examples/pipelines/datacomp/simple_pipeline.py:19]:
```suggestion
```

File: examples/pipelines/datacomp/simple_pipeline.py [hunk 2]
```diff
@@ -27,16 +27,17 @@
...
+    "clip_l14_similarity_score": "imagetext_clipl14score",
 }
 
 load_from_hub_op = ComponentOp(
     component_dir="components/load_from_hf_hub",
     arguments={
-        "dataset_name": "nielsr/datacomp-small-with-embeddings",
+        "dataset_name": "mlfoundations/datacomp_small",
         "column_name_mapping": load_component_column_mapping,
+        # "n_rows_to_load": 100,
```
Comment (RobbeSneyders) [inline examples/pipelines/datacomp/simple_pipeline.py:39]:
```suggestion
        "n_rows_to_load": 100,
```

**Review by RobbeSneyders — APPROVED** [review]

**Pull request merged**